Added Departments tab+extras (#18)
Venture-Roleplay/new-docs: docs/departments/law-enforcement/standard-operating-procedures.mdx

Deputy Commissioner

> Department Supervisors (With the option of Feild trainer (FTO))

Reviewer: Should capitalize "Trainer" and add the word "Officer" to complete the FTO abbreviation
```suggestion
Department Supervisors (With the option of Field Training Officer (FTO))
```

Further explainations based on this topic will be answered in the next section.

#### 2.2 - Duties and responsibilities of the COC

It is importnant to adhere to all responisibilites. Please note this list is not the limit of your duties only a profile of what is expected at VRP.

#### 2.2.1 - Probationary 

As a Probationary you main goals should be to learn as much as possible about how the department runs, as well as the expectations of professionalism at Venture RP's LEO Deaprtment. You should be maintaining a high level of repect for the Feild Training Officers (FTO) as they will be the ones to teach you how to be succsessful in the LEO Department and further, at Venture RP. 

At this stage you will not be allowed to patrol on your own, only with a certified FTO. You must complete a series of FTO ride alongs before you can be promoted to a stand alone rank (Trooper, Officer, Deputy).  

Based on your performance you will be given various opertunities to operate the police vehicle and other patrol duties while acompanied by an FTO. More info on your training will be provided in the training section of this SOP

> <h3 align="center">Section 3</h3>

Reviewer: Should be in the format of the below example for **all** sections
```suggestion
## Section 3 - General Ethics
```

This will improve page indexing. Should only be an `h2` with the double hashtag (`##`)

> <h3 align="center"> The Authority and Jurisdiction of the three Police Departments </h3>

Reviewer: Line should be removed and merged into line 141. Please see comment for 141 for example.
```suggestion
```
